KsApi/lib/TryDecodable.swift

```diff
@@ -0,0 +1,22 @@
+import Argo
+import Foundation
+
+public func tryDecodable<T: Swift.Decodable>(_ json: JSON) -> Argo.Decoded<T> {
```

Comment:
Are we planning to use this in the future? It seems that it's not being used currently.

Reply:
Yeah good call out! I know we don't typically like to do this (and if you feel strongly, I'll remove it), but given that we have a test for it and it's very much our intention to start deprecating Argo using this function I'd like to get it into the codebase. What if I open a PR tomorrow that uses it to deprecate at least one `Argo.Decodable` model? 😄

Reply:
A new PR sounds good. I can review it. Good Job here!